Formula/vdirsyncer.rb

```diff
@@ -1,8 +1,9 @@
+# coding: utf-8
```

Comment:
That is most likely a leftover from me simply copying some other Formula as basis for this one. I'll try to remember to remove it in the next version bump